src/sage/modules/ore_module_morphism.py

```diff
@@ -756,12 +761,26 @@ def kernel(self, names=None):
              m1 + (z+3)*m3 + (z^2+z+4)*m4,
              m2 + (2*z^2+4*z+2)*m4 + (2*z^2+z+1)*m5]
         """
-        ker = self._matrix.left_kernel_matrix()
-        return OreSubmodule(self.domain(), ker, names)
+        mat = self._matrix
+        if hasattr(mat, 'minimal_kernel_matrix'):
+            # Optimisation
+            ker = mat.minimal_kernel_matrix()
+        else:
+            ker = mat.left_kernel_matrix()
+        return OreSubmodule(self.domain(), ker, False, names)
 
-    def image(self, names=None):
+    def image(self, saturate=False, names=None):
         r"""
-        Return ``True`` if this morphism is injective.
+        Return the image of this morphism.
+
+        INPUT:
+
+        - ``saturate`` (default: ``False``) -- a boolean; if ``True``,
+          return the saturation (in the codomain) of the image (see
+          :meth:`saturate` for more details)
```

Comment:
```suggestion
          :meth:`~sage.modules.ore_module.OreSubmodule.saturate` for more
          details)
```
This should link saturate which is in another file.

Reply:
I think there is no `~`.

Reply:
Adding `~` is a feature which removes `sage.modules.ore_module.OreSubmodule` from the printed text, leaving only `saturate` in the displayed link. I do not know if there is any convention in Sage's documentation, so your change without `~` is also fine for me.